kibana/kibana.yml

```diff
@@ -1,67 +1,79 @@
 # Kibana is served by a back end server. This controls which port to use.
-port: 5601
+# server.port: 5601
```

Comment:
Kibana doesn't use a configmap, so it shouldn't be able to be changed from 5601by a user...